### ISSUE TITLE
pre caching of tree isn't the same as on live website

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#11]
Large circular leaves are rendered in fly throughs when the live site somehow knows to precache those parts of the tree and does not render unexpanded parts of the tree as circles

**Comment by jrosindell** [comment]
To clarify what I meant was that at the time of opening this issue the live website did a better job of pre rendering side branches of the tree in advance of a fly through than did the latest version of the software (which is live at the time of writing this comment).  Overall this related to the issue of preloading sufficient data about a fly through before commencing the fly itself so that it's a good experience for the user in terms of both smoothness and detail (this comment related to detail)

**Comment by lentinj** [comment]
I don't think this is really about pre-loading of node details, although obviously more detail would help, I think it's about the fact that it's more obvious. There's 2 factors at play here, all related to nodes not properly knowing if they're children when undeveloped:

* ``branch_layout_helper:draw_interior_circle()`` is dumping a big grey circle on the node end. Polytomy/OTOP both disable this circle to cover node endings since it's not useful with their thin branches. When undeveloped, the radius is wrong and can't be relied upon.
* Undeveloped nodes were assumed to always be leaf nodes, and got a green circle leaf (covering the grey circle). Fixing this is essential for OTOP/Polytomy, but results in barer branches in a standard view (assuming the grey circles are removed).
